requirements.txt

```diff
@@ -1,7 +1,7 @@
 Cython>=0.24
 nilearn>=0.2.4
 numpy>=1.12.1
-scikit-learn>=0.19
+scikit-learn>=1.1.13
```

Comment:
It looks like a typo, I think this should be `1.1.3` instead since there's no published sklearn package with `1.1.13` tag